config/config.go

```diff
@@ -91,6 +91,9 @@ type SnapshotConfig struct {
 type Config struct {
 	m sync.RWMutex `toml:"-"`
 
+	AuthEnabled  bool   `toml:"auth_enabled"`
+	AuthPassword string `toml:"auth_password"`
```

Comment:
Maybe we only need AuthPassword, if it is not empty, we will use password, otherwise, don't. :-)